cli/flox/src/commands/include.rs

```diff
@@ -79,8 +85,15 @@ impl Upgrade {
                 }
             }
         } else {
-            for upgraded in include_diff {
-                message::updated(format!("Upgraded included environment '{upgraded}'"));
+            let mut message = format!("Upgraded {description} with latest changes to:");
+            for upgraded in &include_diff {
+                message.push_str(&format!("\n- '{upgraded}'"));
+            }
```

Comment:
Total style nit, as not functionally relevant,
this `&format!("\n '{upgraded}')` feels. a bit odd, ithink its the prefixed `\n` that tips me off a bit or not using `writeln!(&mut message , "\n '{upgraded}')` or map and join.

Reply:
I thought about using join, but I'd have to join once with `\n- '` and then subsequently with `'\n- '` and then add a closing `'`

If I use `writeline!()`, I'd have to skip the last element cause I don't want a `\n` after the last item

Happy to change it if there's a specific way you have in mind that feels better, but to me both of those options feel more awkward.